README.md

```diff
@@ -30,6 +30,16 @@ Track changes in a [repo](https://gerrit.googlesource.com/git-repo/+/master/#rep
 * `jobs`: *Optional.* number of jobs to run in parallel (default: 0; based on number of CPU cores)
    Reduce this if you observe network errors.
 
+* `insteadOf`: *Optional.* Any URL that starts with this value will be rewritten with the give value.
+   Identical to git url insteadOf option.
+    Example: rewrite http(s):// to git://
+
+    ```yaml
+    insteadOf:
+      "git://": "http://"
+      "git://": "https://"
```

Comment:
why are there 2 identical lines?

Reply:
Lines are different http(S!)
We rewrite http(s):// to git:// in this example